WalletWasabi/Blockchain/TransactionProcessing/ProcessedResult.cs

```diff
@@ -72,7 +72,7 @@ public ProcessedResult(SmartTransaction transaction)
 	/// transaction has successfully invalidated them, because it spends
 	/// some of the same inputs.
 	/// </summary>
-	public List<SmartCoin> SuccessfullyDoubleSpentCoins { get; } = new List<SmartCoin>();
+	public List<SmartTransaction> SuccessfullyDoubleSpentTransactions { get; } = new();
```

Comment:
The comment is outdated here.

Maybe other comments should be updated also.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/11556/commits/c99462db4fe613e7c54d0d95bf837c374354cd36